sw/device/tests/sim_dv/BUILD

```diff
@@ -0,0 +1,174 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan_test.bzl", "opentitan_functest", "verilator_params")
+
+opentitan_functest(
```

Comment:
These tests are only designed to work on the `sim_dv` device because they require a SystemVerilog testbench as the host-side component, so each of the `opentitan_functest` macro should have the arg: `targets = ["dv"]`.